python/ray/dag/py_obj_scanner.py

```diff
@@ -23,6 +23,11 @@
 TransformedType = TypeVar("TransformedType")
 
 
+def delete_instance(id: int):
```

Comment:
as mentioned previously, we don't need this, just explicitly invoke del should work.